admin/users/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url(r'^$', views.UserFormView.as_view(),
+        name='search'),
+    url(r'^id-(?P<guid>[a-z0-9]+)/$', views.UserView.as_view(),
+        name='user'),
+    url(r'^id-(?P<guid>[a-z0-9]+)/rem-2-factor/$', views.remove_2_factor,
```

Comment:
Reversing this action would require a weird refactor of the OSF, essentially storing the two-factor authentication somewhere, and then bringing it back, basically on a whim. Although possibly preferable in the future, this does not help our more immediate concern of actually being able to disable two factor without @saradbowman having to bug @sloria about deleting this addon.